tools/project_manager/show_good_word_suggestions.php

```diff
@@ -37,9 +38,27 @@
 if($format=="update") {
     $postedWords = parse_posted_words($_POST);
 
-    $words = load_project_good_words($projectid);
-    $words = array_merge($words,$postedWords);
-    save_project_good_words($projectid,$words);
+    $is_reject = isset($_POST[REJECT_SUGGESTIONS]);
+    $destination_list = isset($_POST[BAD_WORD_LIST]) ? BAD_WORD_LIST : GOOD_WORD_LIST;
+
+    if ($is_reject)
+    {
+        $postedWords = array();
+    }
+
+    if ($destination_list == GOOD_WORD_LIST)
+    {
+        $words = load_project_good_words($projectid);
+        $words = array_merge($words,$postedWords);
+        save_project_good_words($projectid,$words);
+    }
+    else
+    {
+        var_dump("bad word list");
+        $words = load_project_bad_words($projectid);
+        $words = array_merge($words,$postedWords);
+        save_project_bad_words($projectid,$words);
```

Comment:
Consider moving this outside the if/else block, since it's common to both.

Also, please add spaces after the commas in the function calls (I realize this isn't consistent with the rest of the file).

Reply:
I refactored some, but still needed 2 conditionals for save_project_bad vs save_project_good and load_project_bad vs load_project_good.